foundations/html_css/html-foundations/commit-messages.md

```diff
@@ -0,0 +1,44 @@
+### Introduction
+
+This lesson will explain the importance of good commit messages, how to write them, when to commit and why having a history of good commits is so important!
+
+### Learning Outcomes
+
+By the end of this lesson, you should be able to:
+
+- Write a meaningful commit message
+- Understand why meaningful commit messages are important
+- Have a good understanding of when to commit
+
+
+### Are commit messages so important they deserve their own lesson?
+
+Yes! Let me give you a quick list of reasons why:
+
+- When you are applying for jobs, employers will look through your projects on GitHub and they will look through your commit history. Having good commits as a novice developer will make you stand apart. 
+- Having a good commit message history will allow you (or other developers working on your code) to quickly see what changes were made and why. This is useful if a bug is found in the code that needs to be fixed!
+- Having a good commit message history will also be helpful if you come back to a project you were working on after stepping away from it for a while. You likely won't remember your thought process and changes made when initially writing out the code. 
+
+### How to Write a Good Commit Message
+
+This article, [How to Write a Git Commit Message](https://chris.beams.io/posts/git-commit/),  covers all the main bases on how to write good commit messages. The whole article is great and informative but the meat of the article is “The seven rules of a great commit message.”
```

Comment:
I think this would be better as an assignment rather than just at the top of this section.